machine_manager/models/machinery.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, _
+
+
+class Machinery(models.Model):
+    _name = "machinery"
+    _description = "Manage your machines"
+
+    def _def_company(self):
+        return self.env.user.company_id.id
+
+    name = fields.Char('Machine Name', required=True)
+    company = fields.Many2one('res.company', 'Company', required=True,
+                              default=_def_company)
+    assetacc = fields.Many2one('account.account', string='Asset Account')
+    depracc = fields.Many2one('account.account', string='Depreciation Account')
```

Comment:
From my side, when I see 'anything_id', I know it's a foreign key (an integer with a table behind), not a char field: a huge precious info. When I see 'anything' field I have no information. Why break these conventions ?